mayastor/src/core/bdev.rs

```diff
@@ -115,13 +115,28 @@ impl Share for Bdev {
     }
 
     /// return share URI for nvmf and iscsi
-    fn get_share_uri(&self) -> Option<String> {
+    fn share_uri(&self) -> Option<String> {
         match self.shared() {
             Some(Protocol::Nvmf) => nvmf::get_uri(&self.name()),
             Some(Protocol::Iscsi) => iscsi::get_uri(Side::Nexus, &self.name()),
             None => None,
         }
     }
+
+    fn bdev_uri(&self) -> Option<String> {
```

Comment:
hmm I was indeed torn between adding it here or as a trait when I added the Bdev->Url converter.

Reply:
Yeah arguably, the share trait is not the right place for it and it should be just a method in the Impl block but im not sure how that will playout with replica's later